modules/administration/pages/public-cloud-azure.adoc

```diff
@@ -94,6 +94,8 @@ For more information on setting DNS zones, see the Azure documentation.
 
 Ensure that your {productname} Server is registered with {scc}.
 
+// I fail to se why we need spacewalk-utils!!! What script from this package are we
```

Comment:
@jcayouette git blame says you added this. Why?

Reply:
Looks like it could just be a bit of copypasta?

Reply:
What should we do about this?

Reply:
Ok, let's do the following. Let's keep this PR open until @jcayouette is back tomorrow, so he can merge this and then package.

Reply:
Ya its copy pasta!

Reply:
Ack, but don't resolve yet, as I need to remove it, and you need to review again that I am not breaking anything.

Reply:
I found some more problems.

I think doc squad should really review this whole file and see what makes sense. I don't neither see why we need `spacecmd` or the other packages (others maybe because of the Azure images?)

I will not do this as part of this PR. For `spacewalk-utils` split, we are ready.

Reply:
I will create another issue to track the other changes.